categories.py

```diff
@@ -625,20 +625,33 @@
   ],
   "hlt": [
    "CondFormats/HLTObjects",
-   "CondTools/HLT", "Configuration/HLT",
+   "CondTools/HLT",
+   "Configuration/HLT",
    "CondCore/HLTPlugins",
+   "DataFormats/HLTReco",
    "DQM/HLTEvF",
-   "HLTrigger/Configuration", "HLTrigger/Egamma",
+   "HLTrigger/Configuration",
+   "HLTrigger/Egamma",
    "HLTrigger/HLTanalyzers",
-   "HLTrigger/HLTcore", "HLTrigger/HLTexample",
+   "HLTrigger/HLTcore",
+   "HLTrigger/HLTexample",
    "HLTrigger/HLTfilters",
-   "HLTrigger/JetMET", "HLTrigger/Muon",
+   "HLTrigger/JetMET",
+   "HLTrigger/JSONMonitoring",
+   "HLTrigger/Muon",
    "HLTrigger/Timer",
-   "HLTrigger/Tools", "HLTrigger/btau", "HLTrigger/special",
-   "DataFormats/HLTReco",
-   "RecoTauTag/HLTProducers", "RecoEgamma/EgammaHLTAlgos",
+   "HLTrigger/Tools",
+   "HLTrigger/btau",
+   "HLTrigger/special",
+   "RecoEgamma/EgammaHLTAlgos",
    "RecoEgamma/EgammaHLTProducers",
-   "HLTrigger/JSONMonitoring",
+   "RecoMuon/L2MuonIsolationProducer",
+   "RecoMuon/L2MuonProducer",
+   "RecoMuon/L2MuonSeedGenerator",
+   "RecoMuon/L3MuonIsolationProducer",
+   "RecoMuon/L3MuonProducer",
+   "RecoMuon/L3TrackFinder",
```

Comment:
I think so, RecoMuon/L2* and RecoMuon/L3* are only used for HLT as far as I know.

@sarafiorendi , @BenjaminRS , can you confirm ?